src/index.ts

```diff
@@ -312,6 +331,37 @@ async function prepareApproveERC20Tx(
 	return tx
 }
 
+async function prepareApproveERC721Tx(
```

Comment:
chattiness should be low unless verbose flag is set

I suggest adding an optional verbose param and changing the console logs to `verbose && console.log`

Reply:
Whoops these were debugging statements and weren't really needed anymore